components/samples/context-menu/config-live.js

```diff
@@ -0,0 +1,26 @@
+import {source} from '../../src/source.js';
```

Comment:
This is to make the example use the original MathJax code rather than the webpacked versions (thus the `live` designation).  This allows for debugging without having to continually webpack the changes. The web packing can be done once in the end.  `System.js` can be used on both the web packaged versions and the original MathJax files, so this is not specifically for `system.js`.